### DISCUSSION TITLE
Out of Range Index Warnings

### DISCUSSION THREAD

**Discussion by dieswarte** [springfall2008/batpred#1598] [General]
I was wondering if someone knows how I would go about troubleshooting Out of Range Index warnings in my logs:-

Warn: Out of range index 1 within item idle_end_time value sensor.predbat_GE_1_idle_end_time
Warn: Out of range index 1 within item idle_start_time value sensor.predbat_GE_1_idle_start_time
Warn: Out of range index 1 within item inverter_battery_rate_min value 0
Warn: Out of range index 1 within item device_id value

Still working fine with these warnings, but just out of interest, I was wondering how I can correct those errors.

**Comment by gcoan** [comment]
It's a bug in Predbat,I get the last two as well.

You can ignore them as they are just warnings and predbat works fine, but yes, be better if they didn't happen.
